### ISSUE TITLE
Can't find peers with v0.3.2

### ISSUE THREAD

**Issue by stefa2k** [sigp/lighthouse#1847]
## Description

Beacon isn't able to find peers.

## Version

Docker tag `v0.3.2`has this issue, docker tag `v0.3.1` **works**.

## Present Behaviour

```
beacon_1     | Nov 02 06:51:33.838 WARN Ethereum 2.0 is pre-release. This software is experimental.
beacon_1     | Nov 02 06:51:33.838 INFO Lighthouse started                      version: Lighthouse/v0.3.2-36bd4d87f
beacon_1     | Nov 02 06:51:33.838 INFO Configured for testnet                  name: medalla
beacon_1     | Nov 02 06:51:33.838 INFO Data directory initialised              datadir: /root/.lighthouse/medalla
beacon_1     | Nov 02 06:51:33.876 INFO Starting from known genesis state       service: beacon
beacon_1     | Nov 02 06:51:33.998 INFO Block production enabled                method: json rpc via http, endpoint: http://geth:8545
beacon_1     | Nov 02 06:51:33.998 INFO WebSocket server started                port: 5053, address: 0.0.0.0, service: ws
beacon_1     | Nov 02 06:51:35.586 INFO Beacon chain initialized                head_slot: 0, head_block: 0xebe4…660c, head_state: 0xaf6a…f64e, service: beacon
beacon_1     | Nov 02 06:51:35.586 INFO Timer service started                   service: node_timer
beacon_1     | Nov 02 06:51:35.586 INFO UPnP Attempting to initialise routes    service: UPnP
beacon_1     | Nov 02 06:51:35.587 INFO Libp2p Service                          peer_id: 16Uiu2HAkx8Tm5Xqj3ohaMFKQYx2wJqn1hUZeVA5JmKKtXCHKAuiV, service: libp2p
beacon_1     | Nov 02 06:51:35.587 INFO ENR Initialised                         tcp: Some(9000), udp: None, ip: None, id: 0x362a..02cd, seq: 1, enr: enr:-KO4QE_R8J0oUEnfA22OqbDse7159QcXlXwEhEAAdiJM0xo_anRiwWqiOYhbAOXzxXZnairDE8UtBw2XbQYj5olVnboBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDnp11aAAAAAf__________gmlkgnY0iXNlY3AyNTZrMaECKCpe5wKWS9r824cJVWqgm29j3pOqMJKuc0rmfor9g_6DdGNwgiMo, service: libp2p
beacon_1     | Nov 02 06:51:35.588 INFO Listening established                   address: /ip4/0.0.0.0/tcp/9000/p2p/16Uiu2HAkx8Tm5Xqj3ohaMFKQYx2wJqn1hUZeVA5JmKKtXCHKAuiV, service: libp2p
beacon_1     | Nov 02 06:51:35.604 INFO HTTP API started                        listen_address: 0.0.0.0:5052
beacon_1     | Nov 02 06:51:45.740 INFO UPnP not available                      error: IO error: Resource temporarily unavailable (os error 11), service: UPnP
beacon_1     | Nov 02 06:51:50.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
beacon_1     | Nov 02 06:51:50.001 INFO Searching for peers                     current_slot: 646158, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:02.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:02.001 INFO Searching for peers                     current_slot: 646159, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:14.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:14.002 INFO Searching for peers                     current_slot: 646160, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:26.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:26.001 INFO Searching for peers                     current_slot: 646161, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:38.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:38.001 INFO Searching for peers                     current_slot: 646162, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:50.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
beacon_1     | Nov 02 06:52:50.001 INFO Searching for peers                     current_slot: 646163, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
```

## Expected Behaviour

Having peers, start syncing.

## Steps to resolve

Go back to `v0.3.1`.

**Comment by yorickdowne** [comment]
https://github.com/sigp/lighthouse/pull/1849 should fix that

**Comment by paulhauner** [comment]
I understand this has been resolved in #1849
